src/utils/index.ts

```diff
@@ -37,11 +38,16 @@ export const getEquivalentResource = <Y, D>(
   currentResources: D[],
 ) => {
   if (resource) {
-    const [key, _] = Object.entries(resources ?? []).find((a) => isEqual(a[1], resource))!;
+    const [key, value] = Object.entries(resources ?? []).find((a) => _.isEqual(a[1], resource))!;
     return currentResources.find((e: D) => (e as any).stackResourceId === getResourceID(getStackName(context), key));
   }
 };
 
+export const removeNils = (o: object): object => {
```

Comment:
Let's rename this function to be clear that it does a lot more than removing nils, it also drops any unsupported types